src/status_im/subs/general.cljs

```diff
@@ -292,3 +292,11 @@
  :<- [:toasts]
  (fn [toasts [_ toast-id & cursor]]
    (get-in toasts (into [:toasts toast-id] cursor))))
+
+(re-frame/reg-sub
+ :alert-banners-top-margin
+ :<- [:alert-banners]
+ (fn [banners]
+   (let [banners-count (count banners)]
+     (when (> banners-count 0)
+       (+ (* 40 banners-count) 8)))))
```

Comment:
this `40` needs to be in a constant.
also what is `8` relative to? a named var would help make this easier to understand

Reply:
Hi 40 is banner-height moved to constants.

As I also mentioned in discord, design team adds extra 8 as top margin.

![image](https://github.com/status-im/status-mobile/assets/17097240/ac89a9d7-9de8-4877-bd5f-996cb3839da6)

> a named var would help make this easier to understand

Subscription name is already `alert-banners/top-margin` :thinking: